src/refract/ElementUtils.cc

```diff
@@ -38,7 +38,7 @@ bool refract::inheritsFixed(const IElement& e)
 
 bool refract::inheritsFixed(const ObjectElement& e)
 {
-    return definesValue(e);
```

Comment:
@tjanc is there any reason why this logic requires fixed? I cannot find any cases this change would break, there are no unit/integration tests which cover the case of empty objects and inheriting fixed type attribute.